### DISCUSSION TITLE
Add Lock Or Password To Files Funtionality

### DISCUSSION THREAD

**Discussion by shazm12** [doshareme/src#16] [Ideas]
I propose to have a feature to share files with a lock or password kinda functonality  also so that the sender can make sure even if someone gets a link to his file(which might be highly confidential), the only ones who have that token or password can open the file.

**Comment by vednig** [comment]
Hi @shazm12 we have opened this feature in an issue and are working on it. We will be shipping out many new features, this and upcoming weeks, so keep an eye out for it. #19
